app/src/main/java/com/example/musictoned/addExercise/AddExerciseScreen.kt

```diff
@@ -124,7 +127,7 @@ private fun TopBar(
                 modifier = modifier
                     .fillMaxHeight()
                     .clickable {
-                        onNavigateToEditRoutine()
+                        onNavigateToEditRoutine(null.toString())
```

Comment:
Should this be `exercises[index].name` instead of `null.toString()`? Honestly didn't even know you could do `null.toString` in Kotlin. In TS I know this would throw an exception since `null` doesn't have a `toString` method.